src/github/branch.ts

```diff
@@ -64,14 +59,14 @@ export const processBranch = async (
 	installationId: number,
 	rootLogger: LoggerWithTarget
 ) => {
-	const jiraPayload = await transformBranch(github, webhookPayload);
-
 	const logger = rootLogger.child({
 		webhookId: webhookId,
 		installationId,
 		webhookReceived: webhookReceivedDate
 	});
 
+	const jiraPayload = await transformBranch(github, webhookPayload, jiraHost, installationId, logger);
```

Comment:
I noticed a few of these `transformBranch` calls are sending down information that's needed to create a github client.  I think we should be building the github client and doing the flag check at a high level and passing down the `GithubAPI | GithubClient` so there's no duplication of code across multiple functions, just a simple ternary operator to know which one to use/call.

Reply:
tidied up and only pass through github/githubclient